packages/blade/src/components/Icons/MinusIcon/MinusIcon.tsx

```diff
@@ -0,0 +1,21 @@
+import type { ReactElement } from 'react';
+import { Svg, Path } from '../_Svg';
+import type { IconProps } from '..';
+import useIconProps from '../useIconProps';
+
+const MinusIcon = ({ size, color }: IconProps): ReactElement => {
+  const { height, width, iconColor } = useIconProps({ size, color });
+
+  return (
+    <Svg width={width} height={height} viewBox="0 0 16 2" fill="none">
```

Comment:
Try copying the icon from outermost layer, viewBox seems weird compared to other icons.

Reply:
Modified